packages/runtime-common/module-syntax.ts

```diff
@@ -74,8 +74,11 @@ export class ModuleSyntax {
       | { type: 'exportedName'; name: string }
       | { type: 'localName'; name: string },
     fieldName: string,
-    fieldRef: { name: string; module: string },
+    fieldRef: { name: string; module: string }, // module could be a relative path
     fieldType: FieldType,
+    incomingRelativeTo?: URL,
+    outgoingRelativeTo?: URL,
+    outgoingRealmURL?: URL,
```

Comment:
ingoing and outgoing tends to be confusing imo. Can you add small comment to describe deeper which one is which

Reply:
so the idea is that the module syntax emits code. incoming refers to the parameters being passed into the method, outgoing refers to the code being emitted. the idea being any relative paths in the parameters need to be relative to something, and for relative paths in the emitted code, that also needs to be relative to something.

Reply:
Thanks, I included that info in a code comment